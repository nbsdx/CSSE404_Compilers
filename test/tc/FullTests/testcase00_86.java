class Class1
{
	public static void main (String[] arg)
	{
		boolean z = arg;
	}
}
