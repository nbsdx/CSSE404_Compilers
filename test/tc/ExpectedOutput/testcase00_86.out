Variable arg is not declared.
Cannot assign type null to variable z of type boolean
